File: example/src/main/java/cn/netdiscovery/example/TestSpider4Coroutines.java
package cn.netdiscovery.example;

import cn.netdiscovery.kotlin.coroutines.Spider;
import cn.netdiscovery.kotlin.coroutines.SpiderEngine;

/**
 * Created by tony on 2018/8/14.
 */
public class TestSpider4Coroutines {

    public static void main(String[] args) {

        SpiderEngine spiderEngine = SpiderEngine.create();

        spiderEngine
                .addSpider(Spider.create().name("tony1").url("http://www.163.com"))
                .addSpider(Spider.create().name("tony2").url("http://www.126.com"))
                .addSpider(Spider.create().name("tony3").url("https://www.baidu.com"))
                .httpd(8080)
                .run();
    }
}

File: example/src/main/kotlin/cn/netdiscovery/example/TestSpider4Coroutines.kt
package com.cv4j.netdiscovery.example

import cn.netdiscovery.kotlin.coroutines.Spider

/**
 * Created by tony on 2018/8/14.
 */
fun main(args: Array<String>) {

    Spider.create()
            .name("tony")
            .url("https://www.baidu.com/","https://www.jianshu.com/u/4f2c483c12d8")
            .run()
}